Add `analysis` and `dispersion` to `plasmapy.__init__` (#1512)
PlasmaPy/PlasmaPy: changelog/1512.bugfix.rst

> The `~plasmapy.analysis` and `~plasmapy.dispersion` subpackages are now
> directly imported in `plasmapy.__init__`.

Reviewer: Wrong tense...

```suggestion
Exposed the `~plasmapy.analysis` and `~plasmapy.dispersion` subpackages to the
`plasmapy` namespace..
```

Reviewer: Good catch on this.  I'm at a loss for how this has been missed for so long.

Author: It is indeed import-ant.  😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹

Reviewer: You made the rules, I'm only just enforcing them. 😆
